tests/yam/migration/setup_upgrade.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Makes preprations for migration: version swap, update ttys and notify
+# to not boot from hard disk.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use migration 'reset_consoles_tty';
+
+sub run {
+    # Save the initial/origin version of the product in order to restore later if needed
+    set_var('VERSION_1', get_var('VERSION'));
```

Comment:
VERSION_1 and VERSION_2 don't seem ideal names for me. I would prefer something that anyone outside the team can understand.

Reply:
it simplify the problem of the complicated names given to this, "origin", "original", etc., we are the only ones handling two products, which is a challenge. "2" stands for secondary, and it will not matter if you are in doing migration "from" or "to". The idea is to do a swap of the variables, so the module do not care about it, they will only use SCC_ADDONS or VERSION, not there secondary version. it is hard to find a good name, I will keep an eye I find something better in the future, but I think it helps to be cryptic here for once.